src/options/smt_options.toml

```diff
@@ -207,6 +207,13 @@ header = "options/smt_options.h"
   type       = "bool"
   help       = "after UNSAT/VALID, produce and check an unsat core (expensive)"
 
+[[option]]
```

Comment:
Can you give a quick comment to say this is temporary?

Also, can you just for sanity's sake say how to enable old vs. new unsat cores, as well as what the values of unsatCores, unsatCoresNew, produceProofs are intended to be after set_defaults?

Reply:
haha yes, this is important indeed. Where would be the best place to put this?